packages/native-stack/src/types.tsx

```diff
@@ -155,7 +155,8 @@ export type NativeStackNavigationOptions = {
   headerBackVisible?: boolean;
   /**
    * Title string used by the back button on iOS.
-   * Defaults to the previous scene's title.
+   * Defaults to the previous scene's title, "Back" or arrow icon depending on the available space.
+   * See `headerBackButtonDisplayMode` to read about limitations and customize the behavior.
```

Comment:
I think this is wrong, since react navigation supports this prop also on web and I believe there is no such behaviour as you describe implemented on web. 

What you could do here is either do not modify this description or specify that the "shortening behaviour" applies only to iOS. 

```suggestion
   * Defaults to the previous scene's title.
   * On iOS the text might be shortened to "Back" or arrow icon depending on the available space, following native iOS behaviour.
   * See `headerBackButtonDisplayMode` to read about limitations and interactions with other props. 
```